api/registry/test/test_score_passport.py

```diff
@@ -138,12 +138,16 @@ def __init__(self, account):
                 ),
             )
 
-        with patch("registry.tasks.get_passport", return_value=mock_passport_data):
-            with patch("registry.tasks.async_to_sync", return_value=mock_validate):
+        with patch("registry.atasks.aget_passport", return_value=mock_passport_data):
+            with patch(
+                "registry.atasks.validate_credential", side_effect=mock_validate
+            ):
                 score_passport_passport(self.community.pk, address)
 
+        Passport.objects.get(address=address, community_id=self.community.pk)
+
         score = get_score(mock_request, address, self.community.pk)
-        assert score.score == Decimal("1")
+        assert Decimal(score.score) == Decimal("1")
```

Comment:
Hmmm why is this type conversion now necessary? Did something change about the response type, and if so do we need to account for this downstream anywhere?

Reply:
So, before these changes, we always read the score from the DB before returning.
This was nice, because the result (especially the decimals) where nicely formatted by django as decimals.
Right now we are returning a score that we are computing on-the fly. We do store it in the DB, but we do not reload it from the DB before returning.
Hence the numbers are not formatted nicely. 
By comparing them as `Decimal` we don't care about the formatting, `Decimal("1") == Decimal("1.0000")` will be `True`, and the same is true for something like: `Decimal("0") == Decimal("0e-9")`